trikotFoundation/src/commonTest/kotlin/com/mirego/trikot/foundation/date/DateHelperTest.kt

```diff
@@ -11,16 +12,16 @@ class DateHelperTest {
     fun testDateEquals() {
         val date = Date.now
         assertEquals(false, DateHelper.equals(date, null))
-        assertEquals(false, DateHelper.equals(date, Date.now.plus(1.minutes)))
+        assertEquals(false, DateHelper.equals(date, Date.now.plus(Duration.minutes(1))))
```

Comment:
The `minutes()` extension value doesn't exist anymore?  A shame, seeing `1.minutes` was very easy to understand.

Reply:
It is deprecated (:(), so I took the opportunity to follow the new approach.